src/components/Questionnaire/QuestionnaireEditor.tsx

```diff
@@ -387,11 +390,33 @@ export default function QuestionnaireEditor({ id }: QuestionnaireEditorProps) {
     return true;
   };
 
+  const validateStructuredType = (): boolean => {
+    let hasError = false;
+    const updatedErrors: Record<string, string | undefined> = {};
+
+    questionnaire.questions.forEach((q) => {
+      if (q.type === "structured" && !q.structured_type) {
+        updatedErrors[q.id] = t("select_structure_type");
```

Comment:
let's keep it consistent; it's simply a required field, and all required field errors in care shows this instead of having a dedicated error message specific for each field.

```suggestion
        updatedErrors[q.id] = t("field_required");
```